src/components/layout/Footer.tsx

```diff
@@ -1,18 +1,175 @@
-import { createStyles, makeStyles } from '@material-ui/core'
+import Link from 'next/link'
+import { createStyles, makeStyles, Grid, Container } from '@material-ui/core'
+import { useRouter } from 'next/router'
+import { useTranslation } from 'react-i18next'
+
+import PodkrepiLogo from 'components/brand/PodkrepiLogo'
+import { routes } from 'common/routes'
+
+import { Instagram } from '@material-ui/icons'
+import { Facebook } from '@material-ui/icons'
+import { Twitter } from '@material-ui/icons'
+import { LinkedIn } from '@material-ui/icons'
+import { YouTube } from '@material-ui/icons'
```

Comment:
These can be combined

```suggestion
import { Instagram, Facebook, Twitter, LinkedIn, YouTube } from '@material-ui/icons'
```